assets/app/views/project.html

```diff
@@ -17,7 +17,12 @@
         <div class="row">
           <div class="col-md-12">
             <div class="tile">
-              <h2 class="service osc-object" ng-init="numPorts = service.spec.ports.length" kind="Service" resource="service">{{serviceId}}
+              <div class="route" ng-if="routes[serviceId]">
```

Comment:
could we have multiple routes for the service here? set `class="route osc-object" kind="Route" resource="route"` to get details selection?